.gitignore

```diff
@@ -8,6 +8,9 @@ __pycache__/
 *.fbi
 *.pcap
 *.prof
+.vscode
+.direnv
+.envrc
```

Comment:
No, we don't put IDE-specific rules in gitignore.
(And a number of things should also be removed from this file e.g. *.pcap, *.prof,  .ipynb_checkpoints...)